acceptance/setup/common/pre-suite/010_install_ruby.rb

```diff
@@ -47,6 +47,8 @@
       result = on(bolt, 'ruby --version')
     when /osx/
       # ruby dev tools should be already installed
+      # Pin ffi to 1.9.25 to avoid incompatability with 1.11.1
```

Comment:
Is there an issue we can track?

Reply:
https://github.com/ffi/ffi/issues/702

Reply:
I subscribed to that issue. I am going to merge this for now. Will update if they push a fix.